WordPressKit/AccountServiceRemoteREST.m

```diff
@@ -240,7 +240,8 @@ - (void)requestWPComMagicLinkForEmail:(NSString *)email
     NSMutableDictionary *params = [NSMutableDictionary dictionaryWithDictionary:@{
                                                                                   @"email": email,
                                                                                   @"client_id": clientID,
-                                                                                  @"client_secret": clientSecret
+                                                                                  @"client_secret": clientSecret,
+                                                                                  @"signup_flow_name": @"mobile-ios"
```

Comment:
This method is used both for requesting:

* A magic link to sign in
* A magic link to sign up

I guess we only want the `signup_flow_name` for the last one. Is that right?

If so, we can add an extra parameter and set it here: https://github.com/wordpress-mobile/WordPressKit-iOS/blob/c67da35d2bb75f62cfeeb89fdbce57117588f57c/WordPressKit/AccountServiceRemoteREST.m#L221